app/components/registryBrowse/registryBrowseController.js

```diff
@@ -0,0 +1,40 @@
+angular.module('registryBrowse', [])
+.controller('RegistryBrowseController', ['$q', '$scope', '$transition$', 'RegistryService', 'Notifications',
+function ($q, $scope, $transition$, RegistryService, Notifications) {
+
+  $scope.state = {};
+  $scope.repositories = [];
+
+  function initView() {
+    var registryID = $transition$.params().id;
+
+    RegistryService.catalog(registryID)
+    .then(function success(data) {
+      if (data.headers.link) {
+        Notifications.warning('We only retrieve the first ' + data.data.repositories.length + ' repositories of the registry');
```

Comment:
We don't really like that warning. IMO we should be able to retrieve all repositories, paginating data could be a solution here.

If  we want to keep this at the moment, please specify this as an information directly in the view.

Reply:
Retrieve all data is a bit tricky, cause we don't know how many pages are present. Docker registry api provide only a header with the next url, so we have to loop until we havn't got this header.
I think I'll transform to a simple message for the moment

Reply:
> I think I'll transform to a simple message for the moment

Ok